distributed/asyncio.py

```diff
@@ -0,0 +1,91 @@
+# flake8: noqa
+
+import asyncio
+from functools import wraps
+
+from tornado.gen import is_coroutine_function
+from tornado.platform.asyncio import BaseAsyncIOLoop
+from tornado.platform.asyncio import to_asyncio_future, to_tornado_future
+
+from .client import Client, Future
+
+from tornado.ioloop import IOLoop
+
+
+def to_asyncio(method):
+    @wraps(method)
+    def convert(*args, **kwargs):
+        return to_asyncio_future(method(*args, **kwargs))
+    return convert
+
+
+class AioLoop(BaseAsyncIOLoop):
+
+    @property
+    def _running(self):
+        """Distributed checks IOLoop's _running property extensively"""
+        return self.asyncio_loop.is_running()
```

Comment:
I wouldn't follow this approach, neither the one implemented in `clean_loop` #1021

I'd prefer the following:

```python
import asyncio
from distributed.asyncio import AioClient

explicitly_passed_asyncio_loop = asyncio.get_event_loop()

c = AioClient(loop=loop)
```

This way mixing asyncio and tornado loops can be avoided everywhere except in `distributed.asyncio`